### PR TITLE
Bug 1370594: Rename and optimize Web Docs Logo sprite

### PR DESCRIPTION
**Pull request by stephaniehobson**

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@schalkneethling Added height and width like you suggested, hopefully this fixes the IE problems.

File: kuma/static/img/web-docs-sprite.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+</svg>
```
Comment (schalkneethling) [inline kuma/static/img/web-docs-sprite.svg]: Missing `eol` also, worth running this through SVGOMG. We should probably make SVGO part of the build process as we have more and more SVG assets.

Comment (stephaniehobson) [inline kuma/static/img/web-docs-sprite.svg]: I did run it through SVGOMG.

Comment (stephaniehobson) [inline kuma/static/img/web-docs-sprite.svg]: I don't anticipate adding more than 3 or 4 more SVGs and we only add one or two images a year when we're not redesigning, I'm not sure it's worth updating the build process.

Comment (stephaniehobson) [inline kuma/static/img/web-docs-sprite.svg]: Added the `eol`

**Comment by schalkneethling** [comment]
The SVG is all good, not sure about IE yet. The main reason that I cannot confirm whether it is fixed or not, is because the entire masthead is broken with this PR. I am guessing there is an order in which this and some of the other pull requests needs to be merged?

screen shot 2017-07-11 at 09 04 54

If that is expected r+

**Comment by stephaniehobson** [comment]
Nope, that's definitely wrong :(

**Comment by stephaniehobson** [comment]
Hm. I suspect you still have the CSS from `1370594-home` but the correct HTML from this branch.

**Comment by schalkneethling** [comment]
> Hm. I suspect you still have the CSS from 1370594-home but the correct HTML from this branch.

Indeed, how weird ¯\_(ツ)_/¯

Fixed in IE, all good r+

**Pull request merged**